src/uk/gov/hmcts/contino/Helm.groovy

```diff
@@ -36,19 +36,30 @@ class Helm {
 
   def setup() {
     configureAcr()
+    removeRepo()
     addRepo()
   }
 
   def configureAcr() {
     this.acr.az "configure --defaults acr=${registryName}"
   }
 
+  def removeRepo() {
+    try {
+      this.steps.echo "Clear out helm repo before re-adding"
+      this.steps.sh(script: "helm repo rm ${registryName}")
+    } catch (err) {
+      this.steps.echo "Helm repo may not exist on disk...skipping remove"
+    }
```

Comment:
something like this would be simpler, you may need to update the indenting
```suggestion
      this.steps.echo "Clearing out helm repo before re-adding"
      this.steps.sh(script: "helm repo rm ${registryName} || echo "Failed to remove Helm repository, it probably didn't exist")
```